.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: go
 
 go:
-  - 1.10.3
+  - 1.11.2
 
 before_install:
   - go get -t -v ./ably/...
 env:
   - GOMAXPROCS=4
```

Comment:
please add `GO111MODULE=on` here . Travis clones this in `GOPATH` which disables modules by default.

Travis logs says 

```
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
	see 'go help modules'
```

Reply:
thanks; https://github.com/ably/ably-go/pull/118/commits/ea3f8ab39153a10cd31f99421cb75216c4ec5250